anthos-onprem-terraform/avmw_user_cluster_metallb/main.tf

```diff
@@ -47,12 +47,34 @@ resource "google_project_service" "default" {
   disable_on_destroy = false
 }
 
+# This module is used to update the platform controller on your admin cluster. This
+# is a necessary step for the user cluster version update. If the admin cluster is 
+# already on the correct version, then this module does not change anything
+module "gcloud_update_admin_cluster_platform_controller" {
+  source  = "terraform-google-modules/gcloud/google"
+  version = "~> 2.0"
+
+  platform = "linux"
+
```

Comment:
Remove unnecessary blank lines 56 and 58.

Reply:
Removed. Thanks for checking!